src/d_iwad.c

```diff
@@ -45,9 +45,7 @@ static const iwad_t iwads[] =
     { "freedoom1.wad", doom,     retail,     "Freedoom: Phase 1" },
     { "freedm.wad",   doom2,     commercial, "FreeDM" },
     { "rekkrsa.wad",  doom,      retail,     "REKKR" }, // [crispy] REKKR standalone freeware
-    { "rekkrsa.iwad",  doom,      retail,     "REKKR" }, // [crispy] REKKR standalone freeware non-renamed
-    { "rekkrsl.wad",  doom,      retail,     "REKKR: Sunken Land" }, // [crispy] REKKR Steam retail
-	{ "rekkrsl.iwad",  doom,      retail,     "REKKR: Sunken Land" }, // [crispy] REKKR Steam retail non-renamed
+    { "rekkrsl.wad",  doom,      retail,     "REKKR: Sunken Land" }, // [crispy] REKKR: Sunken Land (Steam retail)
```

Comment:
Does "Sunken Land" even work? AFAIR it was meant to be for GZDoom only, hence the `.iwad` extension.

Reply:
I can confirm that it's simply shipped with GZDoom by default, but that it works perfectly fine in vanilla ports (at least limit-removing ones, haven't tested Choco) if it's run with doom.wad as the IWAD and rekkrsl**.wad** as a PWAD (along with the deh file).

Reply:
@fabiangreffrath Yes, RekkrSL works in Choco as a PWAD `chocolate-doom -iwad doom.wad -merge RekkrSL.wad -dehlump`,
and RekkrSA works both as a PWAD by `chocolate-doom -iwad doom.wad -merge RekkrSA.wad -dehlump` and as an IWAD with the external DEH copied out of the WAD DEHACKED entry by `chocolate-doom -iwad RekkrSA.wad -deh rekkr.deh`.

In Crispy RekkrSA runs even easier, `crispy-doom -iwad RekkrSA.wad`
What I mean to do is to make `crispy-doom -iwad RekkrSL.wad` work.

If that helps, `chocolate-doom -iwad RekkrSL.wad -deh RekkrSL.deh` results in an error message
![RekkrSL_IWAD_error](https://user-images.githubusercontent.com/15119473/186923534-093530f3-2f27-4524-bea0-e854c462b9d3.png)

Reply:
On my end I have no idea why RekkrSL does not run as an IWAD when it does as a PWAD and is Vanilla.
@fabiangreffrath, could you please help me implement it?

Reply:
To keep everything legal you can purchase the game on Steam https://store.steampowered.com/app/1715690/REKKR_Sunken_Land/
It should be cheap even in the EU.